ec2system/ec2machine.go

```diff
@@ -402,6 +428,23 @@ func (s *System) Start(ctx context.Context, count int) ([]*bigmachine.Machine, e
 			securityGroups[i] = aws.String(s.SecurityGroups[i])
 		}
 	}
+
+	// find an ssh key that's know to AWS to ensure that one is available
+	// via instance metadata.
+	var sshKeyName string
```

Comment:
should sshKeyName be a `*string`?  i'm worried that the API will distinguish between an empty string and an absent string. (where the former would fail because there is no key with an empty name)